Apps/Admin/Tests/Functional/cypress/integration/e2e/delegation.cy.js

```diff
@@ -93,3 +94,131 @@ describe("Delegation", () => {
+            .type(dependentWithGuardian.phn);
+        cy.get("[data-testid=search-button]").click();
+
+        // Protect dependent toggle
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "false"
+        );
+
+        // Protect
+        cy.get("[data-testid=dependent-protected-switch]").click();
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "true"
+        );
+        cy.get("[data-testid=cancel-edit-button]").should(
+            "be.visible",
+            "be.enabled"
+        );
+
+        // Save button
+        cy.get("[data-testid=save-button]").click();
+
+        // Confirmation dialog confirmation button
+        cy.get("[data-testid=confirm-button]").should(
+            "be.visible",
+            "be.enabled"
+        );
+
+        // Cancel confirmation
+        cy.get("[data-testid=cancel-button]").click();
+
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "true"
+        );
+
+        // Save button
+        cy.get("[data-testid=save-button]").should("be.visible", "be.enabled");
+
+        // Cancel button
+        cy.get("[data-testid=cancel-edit-button]").click();
+
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "false"
+        );
+    });
+
+    it("Verify protect/unprotect dependent toggle, select to remove, save button and confirmation button are selected.", () => {
+        cy.get("[data-testid=query-input]").clear().type(protectDependent);
+        cy.get("[data-testid=search-button]").click();
+
+        // Confirm delegate table
+        getTableRows("[data-testid=delegate-table]").should("have.length", 2);
+
+        // Protect dependent toggle
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "false"
+        );
+
+        // Protect
+        cy.get("[data-testid=dependent-protected-switch]").click();
+        cy.get("[data-testid=dependent-protected-switch]").should(
+            "have.attr",
+            "aria-checked",
+            "true"
+        );
+
+        // Remove delegate row before protecting dependent
+        const rowSelector =
+            "[data-testid=delegate-table] tbody tr.mud-table-row";
+        cy.get(rowSelector)
+            .first()
+            .within(() => {
+                cy.get("[data-testid=delegate-to-be-removed-checkbox]").click();
+            });
+
+        // Save button
+        cy.get("[data-testid=save-button]").click();
+
+        // Confirmation button
+        cy.get("[data-testid=confirm-button]").click();
+
+        // Confirm delegate table
+        getTableRows("[data-testid=delegate-table]").should("have.length", 2);
```

Comment:
Shouldn't there only be 1 row once the protection is turned on, saved, and confirmed?

Reply:
Changed it.  Test passed with 2 but you're right.